start-from-zero/index.css

```diff
@@ -246,21 +350,208 @@ textarea {
 
 #select {
     float: right;
+    -webkit-appearance: none; 
+    -moz-appearance: none;
+    appearance: none;   
+    background-image: url(assets/down-arrow.png);
+    background-repeat: no-repeat;
+    background-position: 90% 50%;
+    background-size: 13px;
+    position: relative;
+    height: 20px;
+    
+}
+
+
+
+textarea {
+    width: 99.7%;
+    margin: 0 auto;
+    padding: 0;
+}
+
+
+#footer-new {
```

Comment:
Is there a `#footer-old` as well? What other types of `#footer-*` do you support? In most cases, including temporality in your variable names is undesirable, since what is new today might not be all that shiny six months from now. Then, when a new request for change comes along, what do you name that? `#footer-newer`? Then `#footer-newest`? `#footer-even-newer-than-newest`? `#footer-I-hope-this-is-gonna-be-the-last-version-cuz-Im-running-out-of-names`? :P